packages/system/src/globals/global-constants.tsx

```diff
@@ -51,4 +52,13 @@ export const FRAMEWORK_RESOURCES = (bigSize = false): LinkCardProps[] => [
 		backgroundColor: "#5468ff",
 		bigSize,
 	},
+	{
```

Comment:
Lets remove it from here until the page is full of data and deployed, in this way we don't have it in the landing page pointing to a non existing url

Reply:
That makes sense.